taxon_filter.py

```diff
@@ -287,7 +286,6 @@ def parser_deplete_bam_bmtagger(parser=argparse.ArgumentParser()):
                 and db.srprism.idx, db.srprism.map, etc. by srprism mkindex.'''
     )
     parser.add_argument('outBam', help='Output BAM file.')
-    parser.add_argument('--threads', type=int, default=4, help='The number of threads to use in running blastn.')
```

Comment:
Was there a good reason this was 4 previously?

Reply:
I'm not sure. The one place I didn't change the hardcoded default of 4 was for IlluminaBasecallsToSam, which scales memory usage linearly with the number of cores, and can easily exceed JVMmemory on a large flowcell if we're not careful. But I don't think we ever had that issue with blastn...